### DISCUSSION TITLE
Copilot VS 2022: Can't sign out from account

### DISCUSSION THREAD

**Discussion by artificiadrian** [community/community#14252] [Copilot Conversations]
I just downloaded Github Copilot for Visual Studio 2022, and unfortunately signed in with the wrong account.

There is no option to **sign out** in the Visual Studio 2022 version.

Please add that. Reinstalling the extension did not seem to work (I guess credentials are stored somewhere? However I don't know where.)

EDIT: **Temporary solution on Windows**:
Delete the two files (`hosts.json` and `terms.json`) in `C:\Users\%USERNAME%\AppData\Local\github-copilot`

**Comment by liuicl111** [comment]
The same thing happened to me

**Reply by artificiadrian** [reply]
@liuicl111 I found a temporary solution to this problem, at least if you're on **Windows**:

Delete the two files (`hosts.json` and `terms.json`) in `C:\Users\%USERNAME%\AppData\Local\github-copilot`

after that, you can **Sign in** again.

**Reply by liuicl111** [reply]
My problem has been solved. Thank you

**Reply by szybinfover2** [reply]
What if folder `C:\Users\%USERNAME%\AppData\Local\github-copilot` does not exists?

**Reply by opportunityyourway** [reply]
@szybinfover2 same here

**Comment by Zwen092** [comment]
how to solve the same problem on mac?

**Reply by ghost** [reply]
![Screenshot 2022-09-21 at 18 20 52](https://user-images.githubusercontent.com/113331596/191514881-92cbd6c6-5841-4d72-b11f-7ce39777c22b.png)
Left bottom corner of the VS Code window has a button/dropdown that allows the user to sign out

After you sign out successfully, click on the Copilot icon at the right bottom corner and click "retry" if it gives you an error — after that it should prompt a sign in button

**Comment by kzu** [comment]
You can sign out from the same icon where you change all other Copilot settings: 

![image](https://user-images.githubusercontent.com/169707/161850353-f308c70a-a3c0-49bb-b36c-8ec7666d4938.png)

The same menu is accessible from Edit > Copilot suggestions:

![image](https://user-images.githubusercontent.com/169707/161850578-1f677472-af9d-44ea-9845-20d9afdd6c8f.png)

**Reply by artificiadrian** [reply]
This was not the case for me. It still said 'Sign in', even though I was signed in to an account that did not have Github Copilot access.

**Comment by Kundan22** [comment]
in VS2022: From Menu choose File >> Account Settings ; this will show all the logged-in account in VS. remove the Github account to sign-out

**Reply by ewliwtaan** [reply]
Thanks
